lldb/tools/lldb-dap/Handler/InitializeRequestHandler.cpp

```diff
@@ -161,20 +161,22 @@ static void EventThreadFunction(DAP &dap) {
           case lldb::eStateSuspended:
             break;
           case lldb::eStateStopped:
```

Comment:
May be unrelated but in [`lldb_private::StateIsRunningState`](https://github.com/llvm/llvm-project/blob/4b1f1f77b0bd7085ecb8c9a431a3161666ec0f54/lldb/source/Utility/State.cpp#L68) and [`lldb_private::StateIsStoppedState`](https://github.com/llvm/llvm-project/blob/4b1f1f77b0bd7085ecb8c9a431a3161666ec0f54/lldb/source/Utility/State.cpp#L89C6-L89C39) it looks like lldb considers a few other states as 'Running' or 'Stopped', should we match those here?